puppet/modules/leap/manifests/logfile.pp

```diff
@@ -0,0 +1,26 @@
+#
+# make syslog log to a particular file for a particular process.
+#
+
+define leap::logfile($process=$title) {
+  $logfile = "/var/log/leap/${title}.log"
+
+  rsyslog::snippet { "50-${name}":
```

Comment:
well, title and name are not exactly the same. it makes sense for a define called 'logfile' to use title for the log file path, but allow the associated config files to be configurable via $name. but i will make it all $name if you prefer.